jest-ci.config.js

```diff
@@ -1,6 +1,16 @@
 const mainConfig = require("./jest.config");
 
+const testPathIgnorePatterns =
+    process.platform === "darwin" // due to limited resources we do not want to run e2e tests for the maps
+        ? [
+              ...(mainConfig.testPathIgnorePatterns || []),
+              "src/test/e2e-emulated/map-tests/build/*",
+              "src/test/e2e-emulated/getters.spec.ts",
```

Comment:
why do you exclude getters.spec.ts?

Reply:
For some reason we also get a timeout for it:

https://github.com/tact-lang/tact/actions/runs/13195187641/job/36835077839